README.md

```diff
@@ -28,32 +28,56 @@ Logging from the following services is supported for both cases:
       source         = "trussworks/logs/aws"
       s3_bucket_name = "my-company-aws-logs-elb"
       region         = "us-west-2"
-      default_allow = false
-      allow_elb     = true
+      default_allow  = false
+      allow_s3       = true
+      allow_elb      = true
```

Comment:
We're actually specifying 2 services here right?

Reply:
allow_s3 isn't actually specifying any rules in the policy.